hugegraph-dist/docker/example/docker-compose-cassandra.yml

```diff
@@ -0,0 +1,44 @@
+version: "3"
+
+services:
+  graph:
+    image: hugegraph/hugegraph
+    container_name: ca-graph
+    ports:
+      - 18080:8080
+    environment:
+      hugegraph.backend: cassandra
+      hugegraph.serializer: cassandra
+      hugegraph.cassandra.host: ca-cassandra
+      hugegraph.cassandra.port: 9042
+    networks:
+      - ca-network
+    depends_on:
+      - cassandra
+    healthcheck:
+      test: ["CMD", "bin/gremlin-console.sh", "--" ,"-e", "scripts/remote-connect.groovy"]
```

Comment:
we still need `-e`?

Reply:
https://tinkerpop.apache.org/docs/3.6.2-SNAPSHOT/reference/#execution-mode
`-e` is for the gremlin console to use the execution mode to execute scripts.